package.json

```diff
@@ -7,6 +7,7 @@
     "@commitlint/cli": "^12.1.4",
     "@commitlint/config-conventional": "^12.1.4",
     "@opentelemetry/exporter-jaeger": "^0.20.0",
+    "@opentelemetry/api": "^0.20.0",
```

Comment:
I'm still puzzled as to why this breaks only on buildkite, but if it works, let's leave it.

Reply:
Especially puzzling is the fact that `package-lock.json` does not reflect this change.